src/workerinfo.js

```diff
@@ -59,12 +61,43 @@ class WorkerInfo {
     });
   }
 
+  async workerTypeSeen(provisionerId, workerType) {
+    await this.valueSeen(workerType, async () => {
```

Comment:
`workerType` is not globally unique... you can do: `provisionerId + '/' + workerType` as the key... since neither can contain `/`